lib/index.js

```diff
@@ -172,21 +172,18 @@ var OothClient = function () {
             var HEARTBEAT = {
                 ping: 'PING',
                 pong: 'PONG',
-                timeout: 31000
+                timeout: 33000
```

Comment:
Is it ok to make the timeout a little longer? I feel like 1 second  (31 - 30) is a little tight between a ping and the actual timeout.